README.md

```diff
@@ -102,14 +102,27 @@ This command installs everything needed for basic Pipelines that use an Elastics
 
 **Full Installation**
 
-To use more advanced features, like certain DocumentStores, FileConverters, OCR, or Ray, install further dependencies. The following command installs the latest version of Haystack and all its dependencies from the main branch:
+To use more advanced features, like certain DocumentStores, FileConverters, OCR, or Ray, install further dependencies. The following command installs the most recent version of Haystack and all its dependencies from the latest branch:
```

Comment:
Let's remove `from the latest branch`. That's not what we are showing in the pip instal command and it contradicts `most recent version`. If you think it helps, you could mention `latest release` here and maybe even link to the release notes: https://github.com/deepset-ai/haystack/releases

Reply:
To be more consistent with the previous description of the `basic version of Haystack's latest release`, we could write here sth like `full installation of Haystack's latest release with all its dependencies`.